tests/features/support/steps.js

```diff
@@ -462,21 +462,14 @@ When(/I select "([^"]+)" from the ?(?:"([^"]+)")? dropdown/, async (option_text,
 //   }
 // );
 
-Then('I type {string} in the {string} field', async (value, field_label) => {
+Then(/I set the ?(?:"([^"]+)")? text field to "([^"]*)"/, async (field_label, value) => {
   /* Clears the matching textfield in the question area and then types in it */
```

Comment:
When we don't have a labelled text field, we'll have to type `I set the text field to "blah"`. That's slightly different from the previous version of `unlabelled field`. I think it would be good to have an example in the comments specifying the new language.